### PR TITLE
Remove VirusTotal references

### PR DESCRIPTION
**Pull request by JuanGarriuz**
## Description

Removed VirusTotal references

image

### Docs building
- [ ] Compiles without warnings.
### Code formatting and web optimization
- [ ] Uses three spaces indentation.
- [ ] Adds or updates meta descriptions accordingly.
- [ ] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).
### Writing style
- [ ] Uses present tense, active voice, and semi-formal registry.
- [ ] Uses short, simple sentences.
- [ ] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.

### CODE REVIEW

File: source/compliance/nist/malware-detection.rst [hunk 1]
```diff
@@ -11,7 +11,6 @@ Wazuh offers several capabilities that support malware detection. The Malware de
 - File integrity monitoring and threat detection rules.
 - Rootkit behavior detection rules.
 - CDB lists and threat intelligence to detect and remove malicious files
-- VirusTotal integration for malware detection.
```
Comment (guidomodarelli) [inline source/compliance/nist/malware-detection.rst:14]: Do not remove this; it is related to the issue: "Update Malware detection documentation to include VirusTotal."

File: source/compliance/hipaa/malware-detection.rst [hunk 2]
```diff
@@ -9,14 +9,13 @@ Malware detection
...
-- :doc:`Integrating with VirusTotal to detect and remove malware </proof-of-concept-guide/detect-remove-malware-virustotal>`
 - :doc:`Integrating with YARA to detect malware </proof-of-concept-guide/detect-malware-yara-integration>`
 - `Using constant database (CDB) lists to detect and remove malicious files <https://wazuh.com/blog/detecting-and-responding-to-malicious-files-using-cdb-lists-and-active-response/>`__
 
 These components of Wazuh help to comply with the following HIPAA sections:
 
 - **Security Awareness and Training §164.308(a)(5)(i) - Protection from Malicious Software**: *“Procedures for guarding against, detecting, and reporting malicious software.”*
   
-  This section of the HIPAA standard requires you to have procedures to detect and remove malicious software. The Wazuh malware detection capability implements this HIPAA section with the aid of out-of-the-box rules, VirusTotal and YARA integration, and the use of CDB lists. The rootcheck component of Wazuh also detects abnormal behavior in monitored endpoints. These capabilities help support this HIPAA section.
```
Comment (guidomodarelli) [inline source/compliance/hipaa/malware-detection.rst:20]: Do not remove this; it is related to the issue: "Update Malware detection documentation to include VirusTotal."

File: source/getting-started/index.rst [hunk 3]
```diff
@@ -55,9 +55,9 @@ Screenshots
      - .. thumbnail:: ../images/getting-started/configuration-assessment.png
           :title: Security configuration assessment   
           :alt: Security configuration assessment   
-   * - .. thumbnail:: ../images/getting-started/virustotal.png
-          :title: VirusTotal 
-          :alt: VirusTotal 
+   * - .. thumbnail:: ../images/getting-started/endpoints-summary.png
+          :title: Endpoints Summary 
+          :alt: Endpoints Summary
```
Comment (guidomodarelli) [inline source/getting-started/index.rst:60]: Why were these lines changed to "Endpoints summary"?
> [!NOTE]
> The "Endpoints Summary" no longer exists; it has been replaced by "Agents Management » Summary."

File: source/user-manual/wazuh-dashboard/navigating-the-wazuh-dashboard.rst [hunk 4]
```diff
@@ -40,7 +40,6 @@ This section shows dashboards for:
 -  :doc:`Threat Hunting </getting-started/use-cases/threat-hunting>`
 -  :doc:`Vulnerability Detection </user-manual/capabilities/vulnerability-detection/index>`
 -  :doc:`MITRE ATT&CK </user-manual/ruleset/mitre>`
--  :doc:`VirusTotal </user-manual/capabilities/malware-detection/virus-total-integration>`
```
Comment (guidomodarelli) [inline source/user-manual/wazuh-dashboard/navigating-the-wazuh-dashboard.rst:43]: Do not remove this; it is related to the issue: "Update Malware detection documentation to include VirusTotal."

**Review by guidomodarelli — CHANGES_REQUESTED** [review]

**Review by javimed — CHANGES_REQUESTED** [review]
Hi @JuanGarriuz . Please consider this:

- There's a compilation warning about a broken reference that needs to be fixed: `source/user-manual/wazuh-dashboard/navigating-the-wazuh-dashboard.rst:43: WARNING: unknown document: '/user-manual/capabilities/malware-detection/virus-total-integration'`
- Virustotal related documentation stays for older versions. Therefore, you souldn't remove redirectioning in older versions, just adapt redirections concerning 4.10 <-> 4.9. Check the code block below. I can help twith it.
- Your teammates must approve the PR once they consider it's ready for merging.

```
...
/* *** RELEASE 4.10 ****/

/* Redirections from 4.9 to 4.10  */

redirections.push(
  {
    'target': ['4.9=>4.10', '4.10=>4.9'],
    ...
  }

/* Pages no longer available in 4.10 */

removedUrls['4.10'] = [
...
]
...
```

**Review by chantal-kelm — APPROVED** [review]
CR: ✅ 
TEST: ✅ 

Captura de pantalla 2024-11-07 a la(s) 7 16 04 a  m

**Comment by JuanGarriuz** [comment]
> Hi @JuanGarriuz . Please consider this:
> 
> * There's a compilation warning about a broken reference that needs to be fixed: `source/user-manual/wazuh-dashboard/navigating-the-wazuh-dashboard.rst:43: WARNING: unknown document: '/user-manual/capabilities/malware-detection/virus-total-integration'`
> * Virustotal related documentation stays for older versions. Therefore, you souldn't remove redirectioning in older versions, just adapt redirections concerning 4.10 <-> 4.9. Check the code block below. I can help twith it.
> * Your teammates must approve the PR once they consider it's ready for merging.
.....
> ```

After discussing it with the team, those changes will be added to the " Update Malware detection documentation to include VirusTotal" issue.

**Review by javimed — APPROVED** [review]

**Pull request merged**